autokeras/tasks/time_series_forecaster.py

```diff
@@ -98,5 +226,72 @@ def fit(self,
                 validation data.
             **kwargs: Any arguments supported by keras.Model.fit.
         """
-        # TODO: implement.
-        raise NotImplementedError
+        super().fit(x=x,
+                    y=y,
+                    validation_split=validation_split,
+                    validation_data=validation_data,
+                    **kwargs)
+
+    def predict(self, x=None, batch_size=32, **kwargs):
+        """Predict the output for a given testing data.
+
+        # Arguments
+            x: String, numpy.ndarray, pandas.DataFrame or tensorflow.Dataset.
+                Testing data x. If the data is from a csv file, it should be a
+                string specifying the path of the csv file of the testing data.
```

Comment:
I am not sure whether people usually have an `x` to use for the prediction. Maybe some datasets have and some datasets not? We can discuss more during our sync meeting.